lib/tropic/tropic_paragraph.flow

```diff
@@ -584,6 +600,8 @@ positionInteractiveWords(
 	)
 }
 
+wordGroupId = ref 0;
```

Comment:
make it's name more specific, like tropicParagraphWordGroupId and add comment why it is done using global ref